### DISCUSSION TITLE
Git tag messages are not shown by default

### DISCUSSION THREAD

**Discussion by jvortmann** [community/community#6096] [Other Feature Feedback, Questions, & Ideas] [answered]
We use an automated task to create release notes on the git tag message itself from previous git commits trailer information.
Ex.: `git tag v5.1.0 -F <(git shortlog v5.0.0.. --group=trailer:changelog)`

Doing this way, the release notes are available on the repository (on the git tag message) and were used by default on the old version of the release page, allowing us bypass the need to fill any information on the web form.

This new version does not show git tag message by default, instead showing an empty block in place of the notes.

**It is really important to show the tag message by default when no form information is added.**

**Comment by MylesBorins** [comment, marked as answer]
@jvortmann we've identified this as a regression and have a work item to fix this. Thanks for pointing it out!

**Reply by MylesBorins** [reply]
This should be fixed now

**Reply by jvortmann** [reply]
@MylesBorins I'm not seeing the feature preview for that on my user anymore, how can I enable that to test?

**Reply by MylesBorins** [reply]
@jvortmann the updated releases experience is now default, no need to opt-in
